sites/footers/_screen-sm-max.scss

```diff
@@ -3,15 +3,28 @@
   @title: Small view and under (screen only)
  */
 
+/* START: To support footer <= v2.0 */
 #wb-info {
-	.ftr-urlt-lnk {
-		ul {
-			column-count: 2;
-		}
+	.brand {
+		.row {
+			nav {
+				margin: .75em 0 3em;
+
+				ul {
+					column-count: 2;
+					padding-top: 0;
+				}
+
+				li {
+					display: block;
 
-		li {
-			display: block;
-			margin-bottom: .2em;
+					&:first-child::before {
```

Comment:
Comments
I would like to suggest to use the pseudo element `::marker` but it is not fully supported in Safari yet. So we will need to wait at our next major site footer update

FYI
I tested in Safari and we can not set the "content" attribute on a "::marker" pseudo-element only the color which is useless. Also see this Safari bug: https://bugs.webkit.org/show_bug.cgi?id=204163